packages/design-toolkit/src/components/slider/index.tsx

```diff
@@ -127,7 +334,7 @@ export function Slider({
                   )}
                 />
                 <TooltipTrigger
-                  isDisabled={!showInput || state.isThumbDragging(index)}
```

Comment:
the original has `!showInput` which would disable the tooltip when theres NO input, that seems backwards, but i may be wrong.

The tooltip should show the value when there's no input field visible imo, can revert if im wrong though

Reply:
i can remove it and push it in a separate mr, but i did call it out for specifically this reason i said i think i found a bug i stated my logic and i mentioned i can revert if im wrong....